riscv-regs/src/regs.rs

```diff
@@ -117,3 +117,22 @@ impl GeneralPurposeRegisters {
 #[derive(Default)]
 #[repr(C)]
 pub struct FloatingPointRegisters([u64; 32]);
+
+/// The vector register file. We don't expect to directly interact with a guest's vector state
+/// other than for saving/restoring the registers, so simply treat the register file as an array
+/// of 256b values. This actually depends on the vlenb csr, so if the register is greater than 256
+/// bits (i.e. the vlenb csr is greater than 32) we will need to increase this.
+
+// The width of a vector register in bytes
+pub const MAX_VECTOR_REGISTER: usize = 32;
```

Comment:
for some reason, register have width to me, not length.

will fix. Although MAC_VECTOR_REGISTER_LEN makes more sense, since MAX_VECTOR_LEN implies something different

Reply:
fixed